### PR TITLE
Navigator sqlx protos

### PR DESCRIPTION
**Pull request by Ekrekr**
Small PR for protos. Does everything look right?

### CODE REVIEW

File: core/session.ts [hunk 1]
```diff
@@ -81,6 +85,29 @@ function mapColumnDescriptionToProto(
   );
 }
 
+function mapAggregation(aggregation: string) {
+  switch (aggregation) {
```
Comment (BenBirt) [inline core/session.ts:89]: it's so sad we have to do this manually, but protobufjs :(

File: core/session.ts [hunk 2]
```diff
@@ -81,6 +85,29 @@ function mapColumnDescriptionToProto(
...
+    case "sum":
+      return dataform.ColumnDescriptor.Aggregation.SUM;
+    case "distinct":
+      return dataform.ColumnDescriptor.Aggregation.DISTINCT;
+    case "derived":
+      return dataform.ColumnDescriptor.Aggregation.DERIVED;
+    default:
+      return null;
```
Comment (BenBirt) [inline core/session.ts:97]: IMO this should throw

File: core/session.ts [hunk 3]
```diff
@@ -81,6 +85,29 @@ function mapColumnDescriptionToProto(
...
+  }
+}
+function mapDimensionType(dimensionType: string) {
+  switch (dimensionType) {
+    case "category":
+      return dataform.ColumnDescriptor.DimensionType.CATEGORY;
+    case "timestamp":
+      return dataform.ColumnDescriptor.DimensionType.TIMESTAMP;
+    default:
+      return null;
```
Comment (BenBirt) [inline core/session.ts:107]: ditto

File: core/session.ts (hunk 2 above)
Comment (Ekrekr) [reply core/session.ts:97]: Nice idea

File: core/session.ts (hunk 3 above)
Comment (Ekrekr) [reply core/session.ts:107]: ditto

**Pull request merged**